code/parse/sexp.cpp

```diff
@@ -37488,6 +37510,12 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 		"Builtin languages are English, German, French, and Polish, and others can be defined in strings.tbl.\r\n"
 	},
 
+	{ OP_USED_CHEAT, "used-cheat\r\n"
+		"\tReturns true if the give cheat has been used during this mission.  Takes 1 argument.\r\n"
```

Comment:
_And_ there has not been another cheat used since. The SEXP can only test for the last cheat used, and this documentation should account for that